core/src/main/java/org/elasticsearch/transport/TransportService.java

```diff
@@ -71,7 +72,8 @@
 
     public static final String DIRECT_RESPONSE_PROFILE = ".direct";
 
-    private final AtomicBoolean started = new AtomicBoolean(false);
```

Comment:
Can we make this a simple `CountDownLatch` that way we know it's only going in one direction (never move back to blocking), we get the wait for free and don't need to suppress forbidden API?